payload-test.json

```diff
@@ -0,0 +1,146 @@
+{
+    "name": "ami_test_cluster",
+    "releaseLabel": "emr-5.28.0",
```

Comment:
Can we check this is aligned with the ingest hbase version? I dread to think what kind of problems compatibility issues might cause.